inc/components/class-image.php

```diff
@@ -257,7 +257,9 @@ public function configure( $picture ) {
 	 * @return string
 	 */
 	public function get_aspect_ratio( $size_config ) {
```

Comment:
Could add some nullable typehinting to the return value here
```suggestion
	public function get_aspect_ratio( $size_config ): ?string  {
```

Reply:
oo didn't know about these, nice!